docs/global-components/obsolete.md

```diff
@@ -0,0 +1,3 @@
+::: danger These css-docs are obsolete
+# All information about Warp CSS has moved to the [Warp tech-docs](https://warp-ds.github.io/tech-docs/css/).
```

Comment:
```suggestion
# All information about Warp CSS has moved to the [Warp Tech](https://warp-ds.github.io/tech-docs/css/).
```

We've referred to the tech-docs as `Warp Tech` before, so maybe it's worth keeping the same name here, too?

Reply:
Warp tech-docs -> Warp Tech Docs
ok? 🙂